app/views/rooms/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>All Rooms</h1>
+<% @rooms.each do |room| %>
+  <br>
+  Suite: <%= room.suite %>
+  <br>
+  Rate: $<%= room.rate %>.00
+  <br>
+  Hotel: <%= room.hotel.name %>
+  <br>
+  <%= room.suite %>'s Guest Count: <%= room.guest_count %>
+  <br>
+<% end %>
```

Comment:
We mustn't forget about the HTML. Always put rendered ERB in an HTML tag.

```ruby
<p>Suite: <%= room.suite %></p>

<p>Rate: $<%= room.rate %>.00</p>

<p>Hotel: <%= room.hotel.name %></p>

<p><%= room.suite %>'s Guest Count: <%= room.guest_count %></p>
```

This isn't just a plaintext document, this is an HTML file and therefore requires that all tags be surrounded by some HTML element. The HTML element that you plan to use is up to you as long as it is semantically correct.